languages/js/src/Polar.ts

```diff
@@ -314,18 +320,23 @@ export class Polar {
         const ffiQuery = this.#ffiPolar.newQueryFromStr(input);
         const query = new Query(ffiQuery, this.getHost());
         const results = [];
+        let hadResults = false;
+
         for await (const result of query.results) {
-          results.push(result);
-        }
-        if (results.length === 0) {
-          return false;
-        } else {
-          for (const result of results) {
-            for (const [variable, value] of result) {
-              console.log(variable + ' = ' + repr(value));
-            }
+          hadResults = true;
```

Comment:
I updated this to print results as the query executes, but it doesn't quite work correctly if run from within the node repl.